utils/pigz/patches/001.patch

```diff
@@ -0,0 +1,14 @@
+--- a/Makefile	2019-05-22 19:54:23.980451289 +0800
++++ b/Makefile	2019-05-09 14:20:36.269505910 +0800
+@@ -1,4 +1,3 @@
+-CC=gcc
```

Comment:
I don't like this. I would much rather that you replace these 4 variables with ?= instead of =. Otherwise, the build system will not override them.

Reply:
thanks ,done